docs/operation/enable_ip_ip.rst

```diff
@@ -0,0 +1,18 @@
+.. _enable IP-in-IP:
+
+Enable IP-in-IP encapsulation
+=============================
+
+.. _IP-in-IP: https://en.wikipedia.org/wiki/IP_in_IP
+.. _Calico: https://docs.projectcalico.org/
+
+By default Calico_ in MetalK8s is configured to use IP-in-IP_ encapsulation
+only for cross-subnet communication.
+
+To always use IP-in-IP_ encapsulation run the following command:
+
+.. code-block:: shell
+
+    $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
+      patch ippool default-ipv4-ippool --type=merge \
+      --patch '{"spec": {"ipipMode": "Always"}}'
```

Comment:
I think it would make sense to also provide pointers to more in-depth information in our documentation. As an example, we could link to https://docs.projectcalico.org/v3.7/networking/vxlan-ipip from this document.